conf.yaml

```diff
@@ -89,6 +90,19 @@ contents:
               -
                 repo:   kibana
                 path:   docs
+          -
+            title:      Azure Marketplace and Resource Manager (ARM) template
```

Comment:
Would you be open to adding "Deploying the.." to this title?  Then it's less of a change when we eventually roll it into a Deployment Guide?  If yes, I can make that change.

Reply:
I'm good with this. How about "Deploying with..."?